sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -53,10 +53,10 @@ namespace Azure { namespace Core {
 
     /**
      * @brief Get the raw HTTP response.
-     * @return A pointer to #Azure::Core::Http::RawResponse.
+     * @return A #Azure::Core::Http::RawResponse.
      * @note Does not give up ownership of the RawResponse.
      */
-    virtual Azure::Core::Http::RawResponse* GetRawResponse() const = 0;
+    virtual Azure::Core::Http::RawResponse& GetRawResponse() const = 0;
```

Comment:
Add a changelog entry.

Reply:
```suggestion
    virtual Azure::Core::Http::RawResponse const& GetRawResponse() const = 0;
```

The method is marked as `const` but returns a ref to something inside the Operation that we can use to modify the Operation

Reply:
Plus, you may want a non-const version. If so, make the non-const virtual, and implement the const version as non-virtual in this class.

Reply:
I agree with @vhvb1989. I don't think it's necessary for customer to modify the raw request.

Reply:
There is no need for the non-const version as we never expect the contents to be modified by consumers.